lib/ruby_lsp/standard/wraps_built_in_lsp_standardizer.rb

```diff
@@ -0,0 +1,83 @@
+module RubyLsp
+  module Standard
+    class WrapsBuiltinLspStandardizer
+      include RubyLsp::Requests::Support::Formatter
+      def initialize
+        init!
+      end
+
+      def init!
+        @config = ::Standard::BuildsConfig.new.call(ARGV)
+        @standardizer = ::Standard::Lsp::Standardizer.new(
+          @config,
+          ::Standard::Lsp::Logger.new
```

Comment:
Does this logger have any chance to print to stdout? In the LSP context, nothing can be printed to STDOUT or else is breaks editor/server communication. Everything has to be printed to STDERR only.

Reply:
The [Logger](/lib/standard/lsp/logger.rb) is careful to only output over stderr, to keep the stdout communication channel clean